run_snafu.py

```diff
@@ -84,27 +84,26 @@ def main():
             pass
         end_t = time.strftime('%Y-%m-%dT%H:%M:%SGMT', time.gmtime())
 
-    
+
     start_t = datetime.datetime.strptime(start_t, FMT)
     end_t = datetime.datetime.strptime(end_t, FMT)
-    
+
     #get time delta for indexing run
     tdelta = end_t - start_t
     logger.info("Duration of execution - %s" % tdelta)
 
 
 
 def process_generator(index_args, parser):
-    
+
     benchmark_wrapper_object_generator = generate_wrapper_object(index_args, parser)
-    
```

Comment:
I think these whitespace changes are caused by Alex using Eclipse as his editor.   Did you make these changes yourself?

Reply:
Nope i didn't make the changes, so yeah looks like it could be the reason maybe?

Reply:
Yep, i alternate between vim and atom so looks like it